UrbanAirship.Android.Preference.nuspec

```diff
@@ -11,12 +11,13 @@
 
       <dependencies>
          <group targetFramework="MonoAndroid">
-            <dependency id="urbanairship.android.core" version="10.1.1"/>
+            <dependency id="Xamarin.AndroidX.Core" version="1.1.0" />
```

Comment:
what's the dependency on this for? I'm not seeing it referenced anywhere in the preference code.

Reply:
Hmm. I think the AndroidX migration step added Xamarin.AndroidX.Core to the csproj, so I added it here. But you're right, the cs code for the preference package doesn't have any android references. I'll remove the AndroidX packages.